### PR TITLE
tolls

### PR DESCRIPTION
**Pull request by tiruworktito**
**When you open your pull requests, remove the lines that start and end with underscores (\_)**

## Purpose
look at my standard practice

## Approach
added the link to the readme.md

## Pre-Testing TODOs
make sure to test the correct link 
## Testing Steps
_How do the users test this change?_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

_If this closes an issue, name it here. If it doesn't, remove this line_
Closes #6

### CODE REVIEW

**Review by fedluk70 — COMMENTED** [review]
checked

File: README.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 
 ## Contributing to this Repository
 
+* https://www.youtube.com/watch?v=SLB_c_ayRMo
```
Comment (michaelachrisco) [inline README.md:6]: Lets add a description with a note on why this video is important for others to take a look at. That will help everyone else out.

File: README.md [hunk 2]
```diff
@@ -3,6 +3,10 @@
...
+
 #### You are expected to contribute _something_ to our processes. You can do this in many different ways, such as addng documentation that helped you out, writing up a markdown sheet for this repository on a topic you are passionate about, leading a workshop, or posting discussion topics in Bitwise's #devops-apprenticeship-attachment channel.
 
+* tools  https://docs.docker.com/engine/install/linux-postinstall/
```
Comment (michaelachrisco) [inline README.md:10]: Lets add what kind of tools we are looking at here. The link is great! Docker docs are a very good resource. Did you use any specific tool set within the docs?

**Review by michaelachrisco — CHANGES_REQUESTED** [review]
PR looks great @tiruworktito ! I added a couple of changes that I think should be added before this gets merged, just adding descriptions to the links themselves. Awesome job on the PR description as well.

**Pull request closed without merging**